src/terms/terms.jl

```diff
@@ -41,15 +48,9 @@ include("ewald.jl")
 include("psp_correction.jl")
 include("entropy.jl")
 include("magnetic.jl")
-
-# breaks_symmetries on a term builder answers true if this term breaks
-# the symmetries of the lattice/atoms (in which case kpoint reduction
-# is invalid)
-breaks_symmetries(term_builder::Any) = false
 breaks_symmetries(term_builder::Magnetic) = true
-breaks_symmetries(term_builder::ExternalFromReal) = true
-breaks_symmetries(term_builder::ExternalFromFourier) = true
```

Comment:
Actually I liked it more as a block ... first include, then extend.

Reply:
?

Reply:
I do not like your restructuring. I think it's clearer to have the `breaks_symmetries` as a block below the includes.

Reply:
I did it this way because it's attached to each kind of term. When I added the new term I forgot to add the break_symmetry annotation, so I figured it would be clearer this way

Reply:
Ok I see, but it depends what you're after. If you are reading the code and you want to understand what `breaks_symmetries` is doing, than it's better the old way. Question is what happens more often.

Reply:
Less possibility of bugs trumps (very minor) decrease in readability in this case I would say

Reply:
Ok, but then at least make a space after each "include and extend" block